l5kit/l5kit/rasterization/render_context.py

```diff
@@ -24,12 +30,15 @@ def __init__(
         self.raster_size_px = raster_size_px
         self.pixel_size_m = pixel_size_m
         self.center_in_raster_ratio = center_in_raster_ratio
+        self.origin_bottom = origin_bottom
 
         scaling = 1.0 / pixel_size_m  # scaling factor from world to raster space [pixels per meter]
         center_in_raster_px = center_in_raster_ratio * raster_size_px
         self.raster_from_local = np.array(
             [[scaling[0], 0, center_in_raster_px[0]], [0, scaling[1], center_in_raster_px[1]], [0, 0, 1]]
         )
+        if origin_bottom:
+            self.raster_from_local = flip_y_axis(self.raster_from_local, self.raster_size_px[1])
```

Comment:
```suggestion
            self.raster_from_local = vertical_flip(self.raster_from_local, self.raster_size_px[1])
```
I think `vertical_flip` is more clear than `flip_y_axis`: https://www.mathsisfun.com/definitions/vertical-flip.html#:~:text=more%20...,vertical%20direction%20(up%2Ddown)

Reply:
probably, but given I change the name of the variable to `vertical_flip` I can't change this one to the same name

Reply:
How about name the variable as `set_origin_to_bottom ` and rename the function name to `vertical_flip` or `vflip` same as torchvision.transform: https://pytorch.org/docs/stable/torchvision/transforms.html#torchvision.transforms.functional.vflip